python/tskit/tables.py

```diff
@@ -2886,16 +2886,17 @@ def keep_intervals(self, intervals, simplify=True, record_provenance=True):
             in increasing order. The list of intervals must be interpretable as a
             2D numpy array with shape (N, 2), where N is the number of intervals.
         :param bool simplify: If True, run simplify on the tables so that nodes
-            no longer used are discarded. (Default: True).
+            no longer used are discarded. Must be ``False`` if input tree sequence
```

Comment:
I guess this is only because `simplify` itself does not support migrations? So should we note to change this when https://github.com/tskit-dev/tskit/issues/20 is fixed? Perhaps we should add a test of `trim()` with simplify=True on  a ts with migrations, that is currently expected to fail, and comment at the top of the test that once it starts passing, we need to change the docs?